$OpenBSD$

build with system libyuv;  based on work by Vitaly Lipatov <lav@altlinux.ru>
http://sisyphus.ru/ru/srpm/Sisyphus/libowt-tg/patches/1

Index: cmake/libwebrtcbuild.cmake
--- cmake/libwebrtcbuild.cmake.orig
+++ cmake/libwebrtcbuild.cmake
@@ -1,10 +1,6 @@
 add_library(libwebrtcbuild INTERFACE)
 add_library(tg_owt::libwebrtcbuild ALIAS libwebrtcbuild)
 
-target_link_libraries(libwebrtcbuild
-INTERFACE
-    tg_owt::libyuv
-)
 if (NOT absl_FOUND)
     target_link_libraries(libwebrtcbuild INTERFACE tg_owt::libabsl)
 endif()
